docs/topics/testing/advanced.txt

```diff
@@ -67,6 +67,34 @@ The following is a simple unit test using the request factory::
             response = MyView.as_view()(request)
             self.assertEqual(response.status_code, 200)
 
+.. _topics-testing-advanced-allowed-hosts:
+
+Tests and ``ALLOWED_HOSTS``
+===========================
+
+.. versionadded:: 1.11
+
+If your application includes unit tests with custom host names, those tests
+must add the required hosts to :setting:`ALLOWED_HOSTS` using
+:meth:`~django.test.override_settings()` or
+:attr:`~django.test.SimpleTestCase.modify_settings`. This is not common,
+and is usually confined to projects that support multitenancy or otherwise
+alter business logic based on the user's domain (and include tests for that),
+for example::
```

Comment:
This might be worded a bit better. The `, for example` makes me think I'm about to see an example of a project configuration for multi-tenancy, not an example of the now-correct way of testing.

This could be improved by turning the `,` into a `.` and capitalizing the `F` in `for`.